packages/fuel-indexer-benchmarks/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "fuel-indexer-benchmarks"
+version = "0.1.0"
```

Comment:
@lostman If this is a non-publishable crate, this should be `0.0.0`

Here's what's in our `fuel-indexer-tests` manifest.


```toml
name = "fuel-indexer-tests"
version = "0.0.0"
edition = "2021"
license = "BUSL-1.1"
publish = false
```